master/buildbot/test/unit/test_www_oauth.py

```diff
@@ -161,6 +161,11 @@ def test_GithubVerifyCode(self):
                           'username': 'bar',
                           'groups': ["hello", "grp"],
                           'full_name': 'foo bar'}, res)
+        res = yield self.githubAuth.acceptToken("TOK3N")
```

Comment:
I have seen that StopIteration error before, but not lately.  I'm not sure what is causing it.